docs/app/pages/components/sections/panels/item-display-panel-inline/modalDOC.html

```diff
@@ -0,0 +1,5 @@
+<div class="preview-display">
```

Comment:
Would be good to have these as ng-templates in the html just instead. Using templateRefs is preferred over innerHTML (as innerHTML can introduce security vulnerabilities). Take a look at the ngTemplateOutlet which might be useful here